src/domain/Api/index.js

```diff
@@ -42,6 +55,24 @@ export const fetch = (url, options = {}) => {
   });
 };
 
+function getDefaultFetchOpts(options) {
+  const isCorsSimpleMethod =
+      is.empty(options) ||
+      (is.object(options) && (
+        is.undefined(options.method) ||
+        (options.method &&
+          CORS_SIMPLE_METHODS.includes(options.method.toUpperCase()))));
```

Comment:
Maybe we could at least put into a separate function to make it more vertical and readable?